Packs/Akamai_SIEM/ParsingRules/Akamai_WAF_ParsingRules/Akamai_WAF_ParsingRules.xif

```diff
@@ -0,0 +1,3 @@
+[INGEST:vendor="Akamai", product="WAF", target_dataset="akamai_waf_raw", no_hit=keep]
```

Comment:
for existing customers lets take _time out of the code since we already have this

double check this works without special instruction to customers in an upgrade + install from 0 scenarios